### ISSUE TITLE
[Boss] Lich King's normal Frostmourne room.

### ISSUE THREAD

**Issue by Sugarconepk** [WoWManiaUK/Redemption#2242]
I'm reporting an issue regarding healers getting pulled into the frostmourne room and Terenas Menethil http://www.wow-mania.com/armory/?npc=36823 not killing the Spirit Warden http://www.wow-mania.com/armory/?npc=36824

How it works: Whenever a healer gets picked to be pulled into the frostmourne room, healing Terenas and interrupting the soul warden, after a while, neither of them will have died, killing the healer, causing the Lich King to enrage and most likely kill the tank.

How it should work: With the healer inside the frostmourne room, keeping Terenas alive and interrupting the Spirit Warden, Terenas should be able to kill it by himself, preventing the Lich King's soft enrage and teleporting the healer back to the boss fight.

**Comment by fvicaria** [comment]
Nothing has changed in ICC in months so unless this can be reproduced in dev we won't be doing anything about it.

**Comment by Sugarconepk** [comment]
I did indeed test it in dev, and noticed the problem in the first place in redemption.

**Comment by fvicaria** [comment]
Well still can't understand how something like this can be there for the past 2-3 months without anyone noticing.

**Comment by Dranah** [comment]
i'll take a look at this
don't underestimate players' ability to adapt to bugs :P

**Comment by Midnaa** [comment]
This is not true, we had our holy paladin enter and he was able to do the phase just fine. Just because you're a helaer doesen't mean you can't dps while healing.

**Comment by Dranah** [comment]
I found this on wowwiki "The NPC deals nearly enough damage to do the job by himself, but he does need support. "

**Comment by CrustyMuffin** [comment]
Our guild faces that problem too. The moment a healer get´s dragged into it, he instantly dies, even when they have full health. The same thing never happens to any dps or tanks that gets dragged inside.
